### PR TITLE
Fix Broken Links

### PR DESCRIPTION
**Pull request by agan-k**
Dead links for Plone Documentation. Updated with the ones from the [cookie-cutter-plone-starter](https://github.com/collective/cookiecutter-plone-starter?tab=readme-ov-file).

Screenshot 2024-03-13 at 11 22 40 AM

### CODE REVIEW

File: docs/effective-volto/getting-started/project.md
Comment (stevepiercy) [inline docs/effective-volto/getting-started/project.md:32]:
```suggestion
First install `nvm` and latest Node.JS according to the [Plone documentation](https://6.docs.plone.org/install/create-project.html#nvm).
```

File: docs/effective-volto/getting-started/project.md
Comment (stevepiercy) [inline docs/effective-volto/getting-started/project.md:34]:
```suggestion
After that, install Yeoman according to the [Plone documentation](https://6.docs.plone.org/install/create-project.html#yeoman-and-the-volto-boilerplate-generator).
```

File: docs/effective-volto/getting-started/project.md
Comment (stevepiercy) [inline docs/effective-volto/getting-started/project.md:36]:
```suggestion
Finally, install `yarn` according to the [Plone documentation](https://6.docs.plone.org/install/create-project.html#yarn).
```

**Review by stevepiercy — CHANGES_REQUESTED** [review]
Use correct anchors.

**Comment by stevepiercy** [comment]
@agan-k please read and follow https://training.plone.org/contributing/index.html.

**Review by stevepiercy — APPROVED** [review]
LGTM, but we need to wait for your permission to publish before we can merge. Please post a comment when you have been approved. Thank you!

**Comment by stevepiercy** [comment]
@agan-k did you submit a Plone Contributor Agreement?

**Comment by agan-k** [comment]
> @agan-k did you submit a Plone Contributor Agreement?

@stevepiercy I did about 3 days ago.

**Comment by agan-k** [comment]
@stevepiercy the permission just came in.

**Review by stevepiercy — APPROVED** [review]

**Comment by stevepiercy** [comment]
Thank you!

**Pull request merged**
